docs/guides/genomics-platform/requesting-data/about-our-data.md

```diff
@@ -12,30 +12,79 @@ St. Jude Cloud hosts both raw genomic data files and processed results files:
+    * For currently unpublished patient tumor samples, we filtered variants based on metrics derived from the in-house post-processing pipeline and the input of analysts [(see below)](#variant-validation-status). This includes patient tumor samples from the PCGP, G4K, and some Clinical Pilot samples.
+9. Variants were converted from our in-house formats to VCF format.
+10. Variant coordinates were lifted over to GRCh38_no_alt using [Picard][picard] `LiftoverVcf`.
+11. Variants were normalized using [`vt normalize`][vt].
+12. Variants were annotated using [VEP v100][vep] and the `--everything` flag.
+13. VCFs were bgzipped and tabixed, then validated using [VCFtools' `vcf-validator`][vcf-validator].
+    * VCFtools' `vcf-validator` is outdated, and is being replaced with [GATK's `ValidateVariants`][gatk-validator] in future versions of the pipeline.
 
-!!! note
-    **Our Somatic VCF files were designed specifically for St. Jude Cloud visualization purposes. Variants in these files were manually curated from analyses across multiple sequencing types including WGS and WES.**  
-    For more information on variants for each of the individuals, please refer to the relevant PCGP paper. For more information on the variant calling format (VCF), please see the latest specification for VCF document listed [here][hts-specs].
+For more detailed information on steps 1-5, see the [Clinical Pilot paper][clinical-pilot] where they are described in depth.
+
+!!! warning
+    The variants in these files were not called from the HG38 BAMs available elsewhere on St. Jude Cloud, but are the results of a separate HG19 pipeline.
+
+#### Variant Validation Status
+
+When variants are assessed by analysts, they make a determination about how confident they are that the variant is correctly called and present only in the tumor sample. This is a wholistic determination using read counts for all sequencing types available, along with computationally derived statistics created by our post-processing pipeline. Potential pathogenicity is not included in this evaluation.
+
+#### Variant Filtering
+
+We only consider variants for reporting with "VALID", "LIKELY_VALID", or "PUTATIVE" designations (three highest confidence designations) from the variant validation status assessment. This determination is stored in a patient somatic VCF file under the `validation_status` info tag. Note that variants designated "PUTATIVE" are filtered out if they were not computationally determined to be of high quality according to an in-house metric. Any variants which were not manually reviewed by an analyst (no 'validation_status') require this high quality designation to pass filtering. When variants are not manually reviewed, they receive a `validation_status` of "NA" in the VCF.
```

Comment:
I'd like to see a fuller description of what constitutes these three classifications. If I were an external user, based on this description, I'm not sure I'd use these results as I don't have enough information to trust the filtering.

Reply:
I'm not sure how much more information we can provide. The wiki page is pretty sparse on criteria and conversations with Mike R. weren't much more illuminating. We settled on this filtering criteria after everyone we talked to on the ClinGen side repeated this was the best filtering of the data, but I don't think we ever got the details of the designations filled in.

Reply:
I don't think we currently have much documentation on the process that yields these designations. I would like to see that change and this is an opportunity to go in that direction. I'm not sure who is best to document the criteria that lead to these designations.

Reply:
I think Scott Foy may be a good source. We all had a long email chain discussion about this once upon a time when deciding exactly what to do. I'll see if I can find and derive some sense from that. My understanding is that it is a mix of hard rules and judgement calls from the analyst...